.github/workflows/coveo-pyproject.yml

```diff
@@ -1,10 +1,17 @@
 name: coveo-stew
+

```

Comment:
Not very important, be this would have been a good opportunity to rename the file.